sensor_msgs/Image room_map
---
sensor_msgs/Image tesselated_rooms
